seatunnel-e2e/seatunnel-connector-v2-e2e/connector-doris-e2e/src/test/java/org/apache/seatunnel/e2e/connector/doris/DorisIT.java

```diff
@@ -284,8 +284,8 @@ private List<SeaTunnelRow> genDorisTestData(Long nums) {
                                 GenerateTestData.genString(1),
                                 GenerateTestData.genString(11),
                                 GenerateTestData.genString(12),
-                                GenerateTestData.genDatetimeString(false),
                                 GenerateTestData.genDatetimeString(true),
+                                GenerateTestData.genDatetimeString(false),
```

Comment:
Before this PR, the data of generated not match with field order, so the datetime with nano field always insert datetime without nano. So this pr correct it to make sure test work fine.